azure-pipelines-release-dotnet-isolated.yml

```diff
@@ -26,17 +26,17 @@ steps:
   inputs:
     command: restore
     verbosityRestore: Minimal
+    feedsToUse: 'config'
+    nugetConfigPath: 'nuget.config'
```

Comment:
Is this not the default behavior for `nuget restore`?

Reply:
Apparently not! If you look into the build logs, it uses some temp nuget config. We _do_ need to address this eventually though, as we are not supposed to have multiple nuget feeds in our nuget.config.